commands/src/main/java/discord4j/commands/BaseCommand.java

```diff
@@ -0,0 +1,23 @@
+package discord4j.commands;
+
+import discord4j.core.event.domain.message.MessageCreateEvent;
+import reactor.core.publisher.Mono;
+
+/**
+ * Very simple command abstraction.
+ *
+ * @see commands.util.AbstractCommand for a more abstracted command implementation base.
```

Comment:
commands.util.AbstractCommand doesn't exist and prevents the branch from building